scripts/report-bundle-size.js

```diff
@@ -1,21 +1,33 @@
 #!/usr/bin/env node
 /* eslint-env node */
-
+const fs = require('fs')
 const path = require('path')
 const childProc = require('child_process')
 
 /**
  * Send bundle size stats to Datadog
  */
 const main = () => {
-    const retailReactAppStats = require(path.join(path.resolve(''), 'packages', 'template-retail-react-app', 'build', 'stats.json'))
-
-    const bundles = retailReactAppStats.assets
-    bundles.forEach((bundle) => {
-        const metric = `mobify_platform_sdks.bundle_size_byte`
-        const value = bundle.size
-        childProc.spawnSync('dog', ['metric', 'post', metric, value, '--host', bundle.name])
-        console.log(`${metric} ${value} --host ${bundle.name}`)
+    const buildDir = path.join(path.resolve(''), 'packages', 'pwa', 'build')
+    fs.readdir(buildDir, (err, files) => {
+        if (err)
+            console.log(err);
+        else {
+            files.forEach(file => {
+                // bundle report stats files are name as `*-analyzer-stats.json`
+                if (file.includes('-analyzer-stats.json')) {
+                    console.log(`Analyzer stats json file found:`, file)
+                    const retailReactAppStats = require(path.join(path.resolve(''), 'packages', 'pwa', 'build', file))
```

Comment:
We can reuse `buildDir` here. And please make sure that we're looking in the new `template-retail-react-app` package.